.gitignore

```diff
@@ -27,3 +27,5 @@ __pycache__
 .vscode
 .pytest_cache
 .python-version
+venv*/
+.idea/
```

Comment:
Tip: you can put this in your system global gitignore.

Reply:
Good idea @Carreau.